pkg/cluster/cluster.go

```diff
@@ -200,6 +200,45 @@ func (c *Cluster) initUsers() error {
 	return nil
 }
 
+func CreateSecrets() []Action {
```

Comment:
that deserves a comment because I'd expect this function to return sth secret-related and not simply nil